tests/validate-urls.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'English'
+require 'json'
+diff = `git diff --name-only --diff-filter=AM origin/master...HEAD entries/`.split("\n")
+
+def curl(url, path)
+  puts `curl --fail -sSI #{url}`
+  puts "::error file=#{path}:: Unable to reach #{url}" unless $CHILD_STATUS.success?
+end
+
+diff.each do |path|
+  next if path.empty?
+
+  entry = JSON.parse(File.read(path)).values[0]
+
+  if entry.key? 'url'
+    curl(entry['url'], path)
+  else
+    curl("https://#{entry['domain']}/", path)
+    entry['additional-domains'].each { |domain| curl("https://#{domain}/", path) }
+  end
```

Comment:
`additional-domains` should be checked regardless of `url` being present:
```suggestion
  end
  entry['additional-domains'].each { |domain| curl("https://#{domain}/", path) }
```

Reply:
My thought was that it if the `url` tag is set there's probably a high likelihood of `https://#{domain}/` not working as expected on the the `additional-domains` values as well.